test/Kestrel.InMemory.FunctionalTests/Http2/Http2ConnectionTests.cs

```diff
@@ -553,10 +556,10 @@ public async Task DATA_Received_Multiplexed_AppMustNotBlockOtherFrames()
             await StartStreamAsync(1, _browserRequestHeaders, endStream: false);
             await StartStreamAsync(3, _browserRequestHeaders, endStream: false);
 
-            await SendDataAsync(1, _helloBytes, endStream: false);
+            await SendDataAsync(1, _helloBytes, endStream: true);
             Assert.True(stream1Read.WaitOne(TimeSpan.FromSeconds(10)));
 
-            await SendDataAsync(3, _helloBytes, endStream: false);
+            await SendDataAsync(3, _helloBytes, endStream: true);
```

Comment:
Because this test is about overlapping streams, not incomplete streams, so I didn't want it to be affected by the new incomplete streams reset behavior.